data_safe_haven/commands/pulumi.py

```diff
@@ -76,4 +76,4 @@ def run(
         )
 
     stdout = project.run_pulumi_command(command)
-    rprint(stdout)
+    pretty_print(stdout)
```

Comment:
Can we stream this output or are we blocked until the command finishes?

Reply:
I can have a look, I would guess we shouldn't need to worry about that because the OS(?) handles flushing to stdout. I hope `print` returns whether or not the output has been written.